src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,25 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      <UserInfo user={user} />
```

Comment:
The component UserInfo is missing checks for empty user object. It's a good practice to include checks for null or undefined data before rendering components.